### PR TITLE
marsRover23: Adds CSS for the login page

### PR DESCRIPTION
**Pull request by HeatherJLewis**
**Description**
Add CSS to login page. Colour contrast checked on: https://contrastchecker.com/
I think adding SASS would improve the CSS in this file. I've added a ticket to my backlog to add this in the future.

**Changes**
* Adds CSS to the login page

### CODE REVIEW

File: frontend/login.html [hunk 1]
```diff
@@ -10,21 +10,28 @@
...
+                <div class="form-input">
+                    <label for="username">Username</label>
+                    <input type="text" id="username" name="username">
+                </div>
+                <div class="form-input">
+                    <label for="password">Password</label>
+                    <input type="password" id="password" name="password">
+                </div>
+                <div class="form-input">
+                    <input type="submit" name="" value="Login" >
```
Comment (Kenneth-Siu) [inline frontend/login.html]: I don't know if we need a name for the submit button

File: frontend/styles.css [hunk 2]
```diff
@@ -17,23 +38,57 @@ img {
...
+    background-color: #23551b;
+    border: solid 3px #031D40;
+    color: #E0E8ED;
 }
 
-button {
-    margin: 20px;
+.login p a {
+    color: #031D40;
 }
```
Comment (Kenneth-Siu) [inline frontend/styles.css:98]: Did you know that CSS also has constants?

https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

If you're going to switch to sass at some point (which I would want to as well) then there's not really any need, but it's nice that it's there.

Comment (HeatherJLewis) [reply frontend/styles.css:98]: That's really good to know. That's why I wanted to use SASS: to introduce constants.

File: frontend/login.html (hunk 1 above)
Comment (HeatherJLewis) [reply frontend/login.html]: Ok Will remove.

Comment (HeatherJLewis) [reply frontend/login.html]: Done

File: frontend/styles.css (hunk 2 above)
Comment (HeatherJLewis) [reply frontend/styles.css:98]: Just added CSS variables.

**Review by Kenneth-Siu — APPROVED** [review]

**Pull request merged**